src/main/resources/static/javascript/mainApp/grouping.controller.js

```diff
@@ -1753,6 +1753,13 @@
             }
             return str;
         };
+
+        /**
+         * Check if there are no members in the grouping
+         */
+        $scope.isGroupingEmpty = () => {
+            return ($scope.groupingMembers.length === 0 && $scope.groupingBasis.length === 0 && $scope.groupingInclude.length === 0 && $scope.groupingExclude.length === 0);
+        };
```

Comment:
This line is bit long, could you add some newlines at the front of each `&&`? Also the parenthesis can be removed too. Thanks!